### ISSUE TITLE
Make nicknames in markov chain case insensitive

### ISSUE THREAD

**Issue by earthnuker** [alekratz/markov-bot-rs#6]
This would require merging the markov chains that belong to the same nick, maybe run the merging on startup?

**Comment by alekratz** [comment]
Yeah, that's probably behavior we want. Note that there are some other special characters for IRC "lower case", like `\` becomes `|` (or vice versa, I never remember), so you'd have to worry about those, too.
